src/main/java/dk/cngroup/lentils/service/ImageService.java

```diff
@@ -0,0 +1,36 @@
+package dk.cngroup.lentils.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.IOException;
+
+@Service
+public class ImageService {
+
+    public static final String IMAGE_DIRECTORY = "\\images\\";
+
+    public void readAndSaveImageFile(final MultipartFile file) {
+        if (!file.isEmpty()) {
```

Comment:
Napsal bych:

if (file.isEmpty()) {
    return;
}

Reply:
![image](https://user-images.githubusercontent.com/43091754/60011769-cf52d700-967a-11e9-81c4-2ccf8dffca3b.png)

Pokud submitnu bez vybrání souboru.

Reply:
To image view by si myslím nemělo být ve finální verzi.

Reply:
Nn to nebude. Tam pak budou obrazku k tem vecem, co budeme chtit (zatim asi jen hinty, mozna profily tymu).

Reply:
A zaroven bych nedaval jen `return`, ale vyhodil bych vyjimku, ze soubor je neplatny nebo tak neco. Aby klient vedel, co se deje a ze se to nepovedlo.